unity-renderer/Assets/Tutorial/Scripts/Steps/Initial/TutorialStep_GenesisGreetings.cs

```diff
@@ -24,8 +24,15 @@ public class TutorialStep_GenesisGreetings : TutorialStep
 
         public override void OnStepStart()
         {
+            Vector2Int currentCoords = CommonScriptableObjects.playerCoords.Get();
+            if (Mathf.Abs(currentCoords.x) > 3 || Mathf.Abs(currentCoords.y) > 3)
+            {
+                Destroy(gameObject);
```

Comment:
it would be better to terminate the tutorial through a method of the base class instead of destroying the gameObject through a child class, since it could disrupt the flow